flink-table/flink-table-planner/src/main/java/org/apache/flink/table/planner/plan/QueryOperationConverter.java

```diff
@@ -431,21 +428,14 @@ private void splitToProjectionsAndLiterals(
         public RelNode visit(QueryOperation other) {
             if (other instanceof PlannerQueryOperation) {
                 return ((PlannerQueryOperation) other).getCalciteTree();
-            } else if (other instanceof DataStreamQueryOperation) {
-                return convertToDataStreamScan((DataStreamQueryOperation<?>) other);
-            } else if (other instanceof JavaExternalQueryOperation) {
-                final JavaExternalQueryOperation<?> externalQueryOperation =
-                        (JavaExternalQueryOperation<?>) other;
-                return convertToExternalScan(
-                        externalQueryOperation.getIdentifier(),
-                        externalQueryOperation.getDataStream(),
-                        externalQueryOperation.getPhysicalDataType(),
-                        externalQueryOperation.isTopLevelRecord(),
-                        externalQueryOperation.getChangelogMode(),
-                        externalQueryOperation.getResolvedSchema());
-            } else if (other instanceof ScalaExternalQueryOperation) {
-                final ScalaExternalQueryOperation<?> externalQueryOperation =
-                        (ScalaExternalQueryOperation<?>) other;
+            } else if (other
+                    instanceof org.apache.flink.table.planner.operations.DataStreamQueryOperation) {
```

Comment:
call this `InternalDataStreamQueryOperation` so that we can import it? we should drop it at some point.